packages/contentstack-utilities/package.json

```diff
@@ -50,13 +50,14 @@
     "mkdirp": "^1.0.4",
     "open": "^8.4.2",
     "ora": "^5.4.0",
+    "recheck": "^4.4.5",
     "rxjs": "^6.6.7",
     "traverse": "^0.6.7",
     "unique-string": "^2.0.0",
+    "upgrade": "^1.1.0",
```

Comment:
@harshithad0703 , why have you added upgrade library? Any use case?

Reply:
removed this package